README.md

```diff
@@ -8,7 +8,7 @@
 
 ## Setup
 
-The exercises currently target Elixir versions from 1.10 to 1.15 and Erlang/OTP versions from 21 to 26. Detailed installation instructions can be found at
+The exercises currently target Elixir versions from 1.11 to 1.16 and Erlang/OTP versions from 21 to 26. Detailed installation instructions can be found at
 [https://elixir-lang.org/install.html](https://elixir-lang.org/install.html). We recommend using the [asdf version manager](https://github.com/asdf-vm/asdf) to manage multiple Elixir versions.
```

Comment:
At this point I checked the remaining https://elixir-lang.org/ left on the repo and it's all good!